tap_shopify/__init__.py

```diff
@@ -167,7 +167,8 @@ def sync():
             Context.state['bookmarks'] = {}
         Context.state['bookmarks']['currently_sync_stream'] = stream_id
 
-        with Transformer() as transformer:
+        # some fields have epoch-time as date, hence transform into UTC date
+        with Transformer(singer.UNIX_SECONDS_INTEGER_DATETIME_PARSING) as transformer:
```

Comment:
Does this change the structure of existing datetime fields? Were these epoch-time fields previously replicated as epoch-time?

Reply:
> Does this change the structure of existing datetime fields?

No, there is no change in the existing date fields.

> Were these epoch-time fields previously replicated as epoch-time?

No, we have added these fields. There is field named `created` in `order_refunds` whose response in coming in form of epoch time from the API.